articles/resources/relnotes.md

```diff
@@ -17,6 +17,17 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version X.XX.XXXXXXXX
+
+*Release date: November 23rd, 2020*
+
+- Improved compiler performance with faster reference loading.
```

Comment:
Can we list how much faster for typical cases? Do we have any links with more detail, such as descriptions or benchmarks in the relevant PRs themselves?
```suggestion
- Improved compiler performance due to faster reference loading.
```

Reply:
We decided not to publish this data at this time.